bn_mp_pack_size.c

```diff
@@ -0,0 +1,10 @@
+#include "tommath_private.h"
+#ifdef BN_MP_PACK_SIZE_C
+/* LibTomMath, multiple-precision integer library -- Tom St Denis */
+/* SPDX-License-Identifier: Unlicense */
+
+size_t mp_pack_size(mp_int *a)
+{
+   return (size_t)(((mp_count_bits(a) + 1) + (CHAR_BIT - 1)) / CHAR_BIT);
```

Comment:
This should be called mp_pack_count and compute the needed count as computed in mp_pack.

Reply:
count = (bits / ((size * 8u) - nails)) + (((bits % ((size * 8u) - nails)) != 0u) ? 1u : 0u);

Reply:
> This should be called mp_pack_count

Someone&mdash;wasn't it you?&mdash;wanted `mp_pack_size`, so I did.

Reply:
Yes, but I thought a bit about it. If we go with these units of size we should change it. Maxsize should also be maxcount instead? The user should allocate malloc(size*maxcount). countp is also written units.

Alternative would be to go with bytes? countp in bytes, maxsize, pack_size in bytes?

What do you think?